dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
+
+case $TEST_SUITE in
+    unit)
+        cd dev/tests/unit
+        phpunit
+        ;;
+    integration)
+        cd dev/tests/integration
+        phpunit
+        ;;
+    static)
+        cd dev/tests/static
+        phpunit --filter 'Magento\\Test\\Php\\LiveCodeTest::(testCodeStyle|testAnnotationStandard)'
```

Comment:
That would make sense. Would it be acceptable to keep as it is here and expect that when the tests are updated to actually run against the changed files, this can be split again if that performance issue is run into (it would be a very minor change to split them at that point)?

Based on my review, these will only run the static tests on files changed in the current PR, but they are not actually doing anything because the changed file lists aren't being populated, and so I suspect that they may not really take longer than the max job run times when setup correctly.

I'm willing to tackle getting those change lists populating (separate PR, after this is merged in) so we can have them running on travis builds, and if any speed issues are encountered can easily split them.

Let me know your thought, and if you'd rather I just leave them split for now, I can do that too.